Sorting/quick.cpp

```diff
@@ -7,6 +7,33 @@
  *
  * Returns: null
  */
-void Sort::QuickSort(){
-    std::cout<<"[TODO @ Sorting/quick.cpp] QuickSort() needs to be implemented.\n";
+ void Swap(int &a,int &b){
+   int t=a;
+   a=b;
+   b=t;
+}
```

Comment:
put this in sorting.cpp as it is used by many algorithms. You can also use std::swap(&int, &int)

Reply:
To use std::swap(&int, &int), I need to add <algorithm> library in quick.cpp. Am I allowed to do that?

Reply:
I think it'll be better to add the function to sorting.cpp
Including a library is also fine. It'll just increase the compile time a bit.

Reply:
@shashwatj07 Done.